[INFO] ME(225092,7fbb29fb8740,python):2023-04-03-11:05:45.487.258 [mindspore/ccsrc/runtime/hardware/device_context_manager.cc:47] LoadDynamicLib] Load dynamic library: libmindspore_ascend.so failed. libge_runner.so: cannot open shared object file: No such file or directory
[INFO] ME(225092,7fbb29fb8740,python):2023-04-03-11:05:45.525.673 [mindspore/ccsrc/runtime/hardware/device_context_manager.cc:195] LoadPlugin] Load libmindspore_gpu plugin file /home/walter/.local/lib/python3.8/site-packages/mindspore/lib/plugin/libmindspore_gpu.so.11.6 success, skip loading other version.
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:45.540.400 [mindspore/ccsrc/pybind_api/ir/log_adapter_py.h:30] PyExceptionInitializer] Set exception handler
[WARNING] ME(225092:140441839961920,MainProcess):2023-04-03-11:05:45.566.283 [mindspore/run_check/_check_version.py:102] MindSpore version 2.0.0a0 and cuda version 11.4.43 does not match, CUDA version [['10.1', '11.1', '11.6']] are supported by MindSpore officially. Please refer to the installation guide for version matching information: https://www.mindspore.cn/install.
[INFO] CORE(225092,7fbb29fb8740,python):2023-04-03-11:05:45.578.805 [mindspore/core/utils/ms_context.cc:316] SetDeviceTargetFromInner] ms set context device target:GPU
[INFO] PARALLEL(225092,7fbb29fb8740,python):2023-04-03-11:05:45.578.829 [mindspore/ccsrc/frontend/parallel/costmodel_context.cc:30] GetInstance] Create costmodel_context
[INFO] ME(225092:140441839961920,MainProcess):2023-04-03-11:05:45.579.001 [mindspore/run_check/_check_version.py:537] Setting the env `PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python` to prevent memory overflow during save or load checkpoint file.
[WARNING] ME(225092:140441839961920,MainProcess):2023-04-03-11:05:46.307.282 [mindspore/common/api.py:651] 'mindspore.ms_function' will be deprecated and removed in a future version. Please use 'mindspore.jit' instead.
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.324.357 [mindspore/ccsrc/pipeline/jit/pipeline.cc:255] RecordInitStatus] Status record: system init.
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.334.350 [mindspore_federated/fl_arch/ccsrc/common/fl_context.cc:113] set_http_server_address] Http server address is:10.240.17.9:7019
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.343.288 [mindspore_federated/fl_arch/ccsrc/common/fl_context.cc:53] set_server_mode] Server mode: FEDERATED_LEARNING is used for Server and Worker. Scheduler will ignore it.
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.343.311 [mindspore_federated/fl_arch/ccsrc/common/core/yaml_config.cc:148] InitUnsupervisedConfig] cluster_client_num is 0, eval_type is NOT_EVAL
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.343.329 [mindspore_federated/fl_arch/ccsrc/common/core/yaml_config.cc:189] InitCompressionConfig] upload_compress_type is NO_COMPRESS, upload_sparse_rate is 0.4, download_compress_type is NO_COMPRESS
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.343.619 [mindspore_federated/startup/federated_local.py:315] Recovery ckpt files is: [(1, '20230324080920', '/sdb/walter/fedML/federated/example/cross_device_albert/fl_ckpt/Albert_recovery_iteration_1_20230324_080920.ckpt')].
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.343.691 [mindspore_federated/startup/federated_local.py:32] load checkpoint file: /sdb/walter/fedML/federated/example/cross_device_albert/fl_ckpt/Albert_recovery_iteration_1_20230324_080920.ckpt
[INFO] ME(225092:140441839961920,MainProcess):2023-04-03-11:05:46.343.777 [mindspore/train/serialization.py:704] Execute the process of loading checkpoint files.
[INFO] ME(225092:140441839961920,MainProcess):2023-04-03-11:05:46.345.936 [mindspore/train/serialization.py:738] Loading checkpoint files process is finished.
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.346.033 [mindspore_federated/startup/federated_local.py:43] Weight name: conv1.weight, shape: [6, 3, 5, 5], dtype: float32
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.346.098 [mindspore_federated/startup/federated_local.py:43] Weight name: conv2.weight, shape: [16, 6, 5, 5], dtype: float32
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.346.164 [mindspore_federated/startup/federated_local.py:43] Weight name: fc1.bias, shape: [120], dtype: float32
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.346.220 [mindspore_federated/startup/federated_local.py:43] Weight name: fc1.weight, shape: [120, 400], dtype: float32
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.346.274 [mindspore_federated/startup/federated_local.py:43] Weight name: fc2.bias, shape: [84], dtype: float32
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.346.328 [mindspore_federated/startup/federated_local.py:43] Weight name: fc2.weight, shape: [84, 120], dtype: float32
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.346.379 [mindspore_federated/startup/federated_local.py:43] Weight name: fc3.bias, shape: [62], dtype: float32
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.346.431 [mindspore_federated/startup/federated_local.py:43] Weight name: fc3.weight, shape: [62, 84], dtype: float32
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.346.459 [mindspore_federated/startup/federated_local.py:263] Load recovery checkpoint file /sdb/walter/fedML/federated/example/cross_device_albert/fl_ckpt/Albert_recovery_iteration_1_20230324_080920.ckpt successfully.
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.346.494 [mindspore_federated/startup/federated_local.py:328] Recovery iteration num is: 2.
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.346.554 [mindspore_federated/fl_arch/ccsrc/common/fl_context.cc:76] set_ms_role] MS_ROLE of this node is MS_SERVER
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.348.168 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_server.cc:142] Init] The max connection is:10000
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.349.759 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_client.cc:51] CreateBufferEvent] SSL is disable.
[INFO] FEDERATED(225092,7fba88095700,python):2023-04-03-11:05:46.349.936 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_server.cc:264] ListenerCallbackInner] SSL is disable.
[INFO] FEDERATED(225092,7fba87894700,python):2023-04-03-11:05:46.349.946 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_client.cc:175] NotifyConnected] Client connected to the server! Peer Server ip: 10.240.17.9, port: 45443
[INFO] FEDERATED(225092,7fba88095700,python):2023-04-03-11:05:46.349.952 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_server.cc:300] ListenerCallbackInner] A client is connected, fd is 13
[INFO] FEDERATED(225092,7fba87894700,python):2023-04-03-11:05:46.349.965 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_client.cc:208] EventCallbackInner] Client connected! Peer Server ip: 10.240.17.9, port: 45443
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.350.020 [mindspore_federated/fl_arch/ccsrc/common/core/abstract_node.cc:337] InitNodeInfo] The node role:SERVER is generate uuid is:10.240.17.9:45443::2023-04-03-11:05:46.349.997::69383, the ip:10.240.17.9, the port:45443
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.350.030 [mindspore_federated/fl_arch/ccsrc/server/server_node.cc:31] InitializeBeforeCache] [Server start]: Server node create tcp server successful!
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.350.040 [mindspore_federated/fl_arch/ccsrc/common/distributed_cache/redis/redis.cc:675] Init] Try connect to redis sever 10.240.17.9:23456, retry time in seconds 900
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.356.305 [mindspore_federated/fl_arch/ccsrc/common/distributed_cache/redis/redis.cc:689] Init] Connect to redis sever 10.240.17.9:23456 successfully
[WARNING] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.356.355 [mindspore_federated/fl_arch/ccsrc/common/distributed_cache/redis/redis.cc:565] SetExNx] Reply invalid: DENIED Redis is running in protected mode because protected mode is enabled, no bind address was specified, no authentication password is requested to clients. In this mode connections are only accepted from the loopback interface. If you want to connect from external computers to Redis you may adopt one of the following solutions: 1) Just disable protected mode sending the command 'CONFIG SET protected-mode no' from the loopback interface by connecting to Redis from the same host the server is running, however MAKE SURE Redis is not publicly accessible from internet if you do so. Use CONFIG REWRITE to make this change permanent. 2) Alternatively you can just disable the protected mode by editing the Redis configuration file, and setting the protected mode option to 'no', and then restarting the server. 3) If you started the server manually just for testing, restart it with the '--protected-mode no' option. 4) Setup a bind address or an authentication password. NOTE: You only need to do one of the above things in order for the server to start accepting connections from the outside.
[ERROR] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.356.367 [mindspore_federated/fl_arch/ccsrc/server/server.cc:373] InitAndLoadDistributedCache] Sync instance info with distributed cache failed, distributed_cache_address: 10.240.17.9:23456
[WARNING] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.356.419 [mindspore_federated/fl_arch/ccsrc/server/server.cc:79] Run] Catch exception and begin exit, exception: mindspore_federated/fl_arch/ccsrc/server/server.cc:373 InitAndLoadDistributedCache] Sync instance info with distributed cache failed, distributed_cache_address: 10.240.17.9:23456
[INFO] FEDERATED(225092:140441839961920,MainProcess):2023-04-03-11:05:46.356.434 [mindspore_federated/startup/federated_local.py:196] before stopped callback
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.356.477 [mindspore_federated/fl_arch/ccsrc/server/server_node.cc:46] Stop] Begin stop server node!
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.356.486 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_server.cc:204] Stop] Stop tcp server!
[INFO] FEDERATED(225092,7fba88095700,python):2023-04-03-11:05:46.356.545 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_server.cc:185] operator()] Event base dispatch and exit success!
[WARNING] FEDERATED(225092,7fba87894700,python):2023-04-03-11:05:46.356.601 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_client.cc:226] EventCallbackInner] Client connected end of file! Peer Server ip: 10.240.17.9, port: 45443
[INFO] FEDERATED(225092,7fba87894700,python):2023-04-03-11:05:46.356.626 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_client.cc:104] operator()] Event base dispatch failed with no events pending or active!
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.356.638 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_server.cc:204] Stop] Stop tcp server!
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.356.649 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_client.cc:146] Stop] Stop tcp client!
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.356.655 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_client.cc:146] Stop] Stop tcp client!
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.356.660 [mindspore_federated/fl_arch/ccsrc/server/server_node.cc:55] Stop] End stop server node!
Traceback (most recent call last):
  File "./../../run_cloud.py", line 86, in <module>
    start_one_server()
  File "./../../run_cloud.py", line 69, in start_one_server
    job.run(feature_map)
  File "/home/walter/.local/lib/python3.8/site-packages/mindspore_federated/startup/federated_local.py", line 178, in run
    Federated_.start_federated_server(feature_list_cxx, recovery_iteration, self.after_started_callback,
RuntimeError: mindspore_federated/fl_arch/ccsrc/server/server.cc:373 InitAndLoadDistributedCache] Sync instance info with distributed cache failed, distributed_cache_address: 10.240.17.9:23456
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.236 [mindspore/ccsrc/pipeline/jit/init.cc:431] operator()] Start releasing dataset handles...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.255 [mindspore/ccsrc/pipeline/jit/init.cc:434] operator()] End release dataset handles.
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.264 [mindspore/ccsrc/pipeline/jit/pipeline.cc:2010] ClearResAtexit] Pipeline clear all resource
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.384 [mindspore/ccsrc/pipeline/jit/pipeline.cc:260] RecordExitStatus] Status record: system exit.
[INFO] DEBUG(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.405 [mindspore/ccsrc/common/debug/env_config_parser.cc:153] ParseFromFile] The 'env_config_path' in 'mindspore.context.set_context(env_config_path={path})' is empty.
[INFO] ME(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.554 [mindspore/core/mindrt/src/actor/actormgr.cc:150] Finalize] mindrt Actors finish exiting.
[INFO] ME(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.565 [mindspore/core/mindrt/src/actor/actormgr.cc:153] Finalize] mindrt Threads finish exiting.
[INFO] ME(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.571 [mindspore/core/mindrt/src/actor/actormgr.cc:164] Finalize] mindrt IOMGRS finish exiting.
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.579 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1892] ClearResPart1] Start clear kernel runtime...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.588 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1894] ClearResPart1] End clear kernel runtime.
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.593 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1896] ClearResPart1] Start Finalize StreamSynchronizer...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.375.601 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1898] ClearResPart1] End Finalize StreamSynchronizer...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.832 [mindspore/ccsrc/pipeline/jit/pipeline.cc:654] ClearRes] Clean executor resource!
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.841 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1914] ClearResPart2] Start clear PyNativeExecutor...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.862 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1916] ClearResPart2] End clear PyNativeExecutor.
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.869 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1928] ClearResPart2] Start clear ConfigManager...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.874 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1930] ClearResPart2] End clear ConfigManager.
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.879 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1937] ClearResPart2] Start clear device context...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.886 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1939] ClearResPart2] End clear device context.
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.891 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1941] ClearResPart2] Start clear AnalysisResultCacheMgr...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.897 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1943] ClearResPart2] End clear AnalysisResultCacheMgr.
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.901 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1945] ClearResPart2] Start clear AnalysisContext...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.908 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1947] ClearResPart2] End clear AnalysisContext...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.913 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1949] ClearResPart2] Start clear AnalysisSchedule...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.938 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1951] ClearResPart2] End clear AnalysisSchedule...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.945 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1954] ClearResPart2] Start clear python_adapter...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.950 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1956] ClearResPart2] End clear python_adapter.
[INFO] DEBUG(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.962 [mindspore/ccsrc/debug/debugger/debugger.cc:95] Debugger] Debugger got device_target: GPU
[INFO] DEBUG(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.970 [mindspore/ccsrc/debug/debugger/debugger.cc:299] Reset] Release Debugger resource.
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.975 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1966] ClearResPart3] Start clear ClearObjectCache...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.980 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1968] ClearResPart3] End clear ClearObjectCache...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.985 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1970] ClearResPart3] Start clear Parser...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.991 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1972] ClearResPart3] End clear Parser...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.376.996 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1974] ClearResPart3] Start ClearTraceStack...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.377.006 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1976] ClearResPart3] End ClearTraceStack...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.377.012 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1978] ClearResPart3] Start clear InterpretNodeRecorder...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.377.017 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1980] ClearResPart3] End clear InterpretNodeRecorder...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.377.022 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1982] ClearResPart3] Start clear parallel::entire_costgraph...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.377.029 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1984] ClearResPart3] End clear parallel::entire_costgraph...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.377.034 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1986] ClearResPart3] Start clear ProtobufLibrary...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.377.104 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1988] ClearResPart3] End clear ProtobufLibrary...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.377.112 [mindspore/ccsrc/pipeline/jit/pipeline.cc:1992] ClearSingleton] Start clear singleton...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.377.156 [mindspore/ccsrc/pipeline/jit/pipeline.cc:2006] ClearSingleton] End clear singleton.
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.377.161 [mindspore/ccsrc/pipeline/jit/pipeline.cc:2015] ClearResAtexit] Start unload dynamic lib...
[INFO] PIPELINE(225092,7fbb29fb8740,python):2023-04-03-11:05:46.377.170 [mindspore/ccsrc/pipeline/jit/pipeline.cc:2017] ClearResAtexit] End unload dynamic lib...
[INFO] FEDERATED(225092,7fbb29fb8740,python):2023-04-03-11:05:46.499.289 [mindspore_federated/fl_arch/ccsrc/common/communicator/tcp_client.cc:72] Stop] Stop tcp client event dispatch!
